app/helpers/hosts_helper.rb

```diff
@@ -183,7 +183,7 @@ def multiple_actions_select
   end
 
   def date(ts = nil)
-    return _("%s ago") % (time_ago_in_words ts) if ts
+    return (date_time_relative ts) if ts
     _("N/A")
```

Comment:
I prefer this style
```suggestion
    return date_time_relative(ts) if ts
    _("N/A")
```

Or

```suggestion
    ts ? date_time_relative(ts) : _("N/A")
```